### PR TITLE
feat(examples): add example React app to demonstrate how to use the components 

### LINKED ISSUE #133: Provide a simple React example to verify that the package can be used in JS code

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #133

### PR Checklist

- [ ] The implemented feature is covered by an appropriate test.

### CODE REVIEW

File: examples/React/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Install the packed version of the components:
+
+```bash
+npm run import-components
+```
+
+Start a test server to see the components in action:
+
+```bash
+npm run start
```
Comment (JonasKellerer) [inline examples/React/README.md:14]: This seems to be
```suggestion
npm run dev
```
?

Comment (JonasKellerer) [reply examples/React/README.md:14]: How are the other commands used from the `package.json`?

File: examples/React/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+```
+
+Then, open http://localhost:5173/ in your browser.
```
Comment (JonasKellerer) [inline examples/React/README.md:17]: For me this opens in localhost:5174 ?

File: examples/React/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        if (locationFilter) {
+            locationFilter.addEventListener('gs-location-changed', handleLocationChange);
+        }
+
+        const dateRangeSelector = document.querySelector('gs-date-range-selector');
+        if (dateRangeSelector) {
+            dateRangeSelector.addEventListener('gs-date-range-changed', handleDateRangeChange);
+        }
+
+        // Cleanup function to remove event listeners
```
Comment (JonasKellerer) [inline examples/React/src/App.tsx:29]: Do we need this comment?

**Review by JonasKellerer — APPROVED** [review]
LGTM

Only the README needs some care.

File: examples/React/README.md (hunk 2 above)
Comment (fengelniederhammer) [reply examples/React/README.md:17]: Maybe we should simply get rid of this line. vite tells you which url to open and it looks for a free port (if you already have another `vite dev` running that block port 5173)

File: examples/React/README.md (hunk 1 above)
Comment (fengelniederhammer) [reply examples/React/README.md:14]: They are not used. I deleted them.

**Pull request merged**